### PR TITLE
update statute style

### PR DESCRIPTION
**Pull request by porta-antiporta**
Makes the following changes
- fixes bug on Regulations style that wasn’t highlighting regulation name
- makes Statutes consistent with Regulations style

### CODE REVIEW

File: openfecwebapp/templates/partials/legal-search-results-statute.html [hunk 1]
```diff
@@ -1,18 +1,14 @@
 <div class="simple-table simple-table--responsive simple-table--display legal-search-results">
```
Comment (porta-antiporta) [inline openfecwebapp/templates/partials/legal-search-results-statute.html]: @adborden couldn't find `legal-search-results` class in fec-style.  Not sure what this was for.

**Comment by porta-antiporta** [comment]
fixes https://github.com/18F/fec-eregs/issues/239

File: openfecwebapp/templates/partials/legal-search-results-statute.html (hunk 1 above)
Comment (adborden) [inline openfecwebapp/templates/partials/legal-search-results-statute.html]: Good catch, it's an old style and I think it can be removed.

**Comment by adborden** [comment]
:+1: looks good, but I'm not going to hit the merge button until after the demo, but don't let that stop anyone else.

**Pull request merged**